src/Routes.js

```diff
@@ -2,19 +2,18 @@
 const routes = {
   HomePage: {
     url: '/',
-    component: 'src/pages/HomePage.js',
-    seo: {
-      title: 'Home',
-      description: '',
-    },
+    component: 'src/pagesComponents/HomePage.js',
```

Comment:
Why did this path change? I actually like `src/pages/...` better for the roots of the pages.

Reply:
I changed it because of pages duplicate. 

By default, for each files in `pages/` a new page is created with file's name as url. 
As I was customizing the urls in `Routes.js`, we got 2 pages with different urls

Others solutions can be 
-  to import some package and try to change the default created pages urls
-  to rename the files in `pages/` let them match with our urls

@bencpeters

Reply:
I actually like the idea of keeping things simple - I think matching URLs with filenames for stuff in `pages/` makes sense and is probably the path of least resistance.